xcube/cli/io.py

```diff
@@ -172,38 +174,79 @@ def writer_info(writer_id: str):
     if description:
         print(description)
     from xcube.core.store import new_data_writer
-    writer = new_data_writer(writer_id)
-    params_schema = writer.get_write_data_params_schema()
+    writer_ = new_data_writer(writer_id)
+    params_schema = writer_.get_write_data_params_schema()
     print(_format_params_schema(params_schema))
 
 
 @click.command(name='dump')
-@click.option('-o', '--output', 'file_path', help='Output filename.', default='store-dump.json')
-def dump(file_path: str):
+@click.option('-o', '--output', 'output_file_path', metavar='OUTPUT',
+              help='Output filename. Output will be written as JSON.', default='store-dump.json')
+@click.option('-c', '--config', 'config_file_path', metavar='CONFIG',
+              help='Store configuration filename. May use JSON or YAML format.')
+@click.option('-t', '--type', 'type_specifier', metavar='TYPE',
+              help='Type specifier. If given, only data resources that satisfy the '
+                   'type specifier are listed. E.g. "dataset" or "dataset[cube]"')
+def dump(output_file_path: str, config_file_path: Optional[str], type_specifier: Optional[str]):
     """
-    Dump all store metadata.
-    Dumps all store metadata and their dataset descriptors into a single JSON file.
+    Dump metadata of given data stores.
+    Dumps metadata for every data resources for given data stores.
```

Comment:
Alternative Suggestion:
"Dump metadata of data store resources
Dumps metadata of every data resource of available data stores.
A restriction on which data stores are considered may be passed via the CONFIG option."

If you don't approve, please at least correct "for every data resources".

Reply:
> If you don't approve, please at least correct "for every data resources".

I believe "for every data resources" is no good English.
I'll try to improve docs.